widget_driver/example/example.md

```diff
@@ -38,28 +38,21 @@ part 'my_first_drivable_widget_driver.g.dart';
 class MyFirstDrivableWidgetDriver extends WidgetDriver {
   int _count = 0;
 
-  @TestDriverDefaultValue('The app bar title')
```

Comment:
Perhaps we could leave this one TestDriverDefaultValue in and add a comment that adding this Annotation is optional here (as can be seen in the next variable) but still is possible. Just to highlight new users that they still could set a default value.

Reply:
Nice ideas @dominik089